### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by akurdunkar** [akurdunkar/gspt#16] [Ideas]
## These are the current features that are part of the roadmap

- [x] Rounded Corners
- [x] Key Mappings
- [x] Queue Support
- [ ] Multiple Image rendering backends
    - [ ] Sixel
    - [ ] tview Images
    - [ ] Kitty Images
- [ ] Customisable UI
- [ ] Wayland Support for Image Rendering
- [ ] Support for Podcasts

**Comment by benmanone** [comment]
Will the app ever support podcasts?

**Reply by akurdunkar** [reply]
Yes. Should probably add this to the Roadmap
